### PR TITLE
Update @codeboten affiliation to reality

### PR DESCRIPTION
**Pull request by bogdandrutu**
Fix the denial that @codeboten lives in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,7 +144,7 @@ Here is a list of community roles with current and previous members:
 
 - Maintainers ([@open-telemetry/collector-maintainers](https://github.com/orgs/open-telemetry/teams/collector-maintainers)):
 
-   - [Alex Boten](https://github.com/codeboten), Lightstep
+   - [Alex Boten](https://github.com/codeboten), ServiceNow
```
Comment (codeboten) [inline README.md:147]:
```suggestion
   - [Alex Boten](https://github.com/codeboten), ServiceNow
```

**Review by codeboten — CHANGES_REQUESTED** [review]

**Comment by bogdandrutu** [comment]
PTAL @codeboten

File: README.md [hunk 2]
```diff
@@ -144,7 +144,7 @@ Here is a list of community roles with current and previous members:
...
```
Comment (yurishkuro) [inline README.md:147]: harsh...

Comment (bogdandrutu) [reply README.md:147]: @tedsuo what do you think?

**Review by codeboten — APPROVED** [review]

**Pull request merged**